manual/python/function.md

```diff
@@ -6,15 +6,17 @@ You can use supported functions in SQL query statements.
 
 With functions you can transform, calculate, combine or merge the values of other columns from the current table. On top of that, functions can refer to each other. 
 
+The functions supported in SQL are roughly the same as the set of functions supported by formulas in SeaTable.
+
 The basic syntax of functions is as follows:
 
 ```
 FunctionName(parameters...)
-The parameters can be number,string,constants,column name or other functions.
-
 
 ```
 
+The parameters can be number,string,constants,column name or other functions.The quoted column name cannot be an alias. If the column name contains "-", you can use "\`" to widen it.
```

Comment:
The parameters can be number,string,constants,column name or other functions. Column name cannot be an alias. If the column name contains "-", you can use "\`" to enclose it.